server/util/redact/redact.go

```diff
@@ -0,0 +1,79 @@
+package redact
+
+import (
+	"context"
+	"regexp"
+	"strings"
+
+	"github.com/buildbuddy-io/buildbuddy/server/environment"
+	"github.com/golang/protobuf/proto"
+
+	bespb "github.com/buildbuddy-io/buildbuddy/proto/build_event_stream"
+)
+
+const (
+	RedactionFlagAPIKey = 1 << iota
```

Comment:
Since this is written into the database -- can you explicitly define it?

Reply:
Done